editor.planx.uk/src/@planx/components/shared/Preview/MapContainer.tsx

```diff
@@ -8,31 +8,18 @@ interface MapContainerProps {
   size?: "large";
 }
 
-/**
- * Generate a style which increases the map size as the window grows
- * and maintains a consistent right margin
- */
-const dynamicMapSizeStyle = (theme: Theme): Record<string, any> => {
```

Comment:
I picked up this as the `theme` argument was flagging as unused - it made sense just to tidy it up.